.github/workflows/workflow-generate-ci-documentation.yml

```diff
@@ -0,0 +1,64 @@
+name: Generate CI Documentation
+on:
+  workflow_call:
+    inputs:
+      commit_resources:
+        description: 'Commit the resources after generating them. (Requires the access_token to be passed)'
+        required: false
+        default: false
+        type: boolean
+    secrets:
+      access_token:
+        description: 'The access token to use for commits.'
+        required: false
+  workflow_dispatch:
+    branches:
+    - main
+    - develop
+    - "feature-*"
+    - "release-*"
+    inputs:
+      commit_resources:
+        description: 'commit the resources after generating them. Requires a PAT defined as secrets.COMMIT_TOKEN'
+        required: true
+        default: false
+        type: boolean
+jobs:
+  update_ci_documentation:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Generate CI Documentation
+        uses: Compton-NIST/Inspector@v1
+        with:
+          artifact_name: docs_ci
+          output_path: docs_ci
+          file_prefix: OSCAL
+      - name: Make Overview the README.md for the directory.
+        run: |
+          cp docs_ci/OSCAL.workflows.md docs_ci/README.md
+          rm -f docs_ci/OSCAL.workflows.md
+      - name: Commit Documentation Result (Documents and Diagrams)
+        if: github.event.inputs.commit_resources == 'true' || inputs.commit_resources == true
+        uses: stefanzweifel/git-auto-commit-action@3ea6ae190baf489ba007f7c92608f33ce20ef04a
+        with:
+          create_branch: true
+          branch: auto-ci-docs-update
```

Comment:
Non-blocking: I know we discussed this in passing, but the intent is always to add this to one specific branch, and not the target branch of where the PR will run. Am I understanding correctly: we are going to add this and run from any PR into a target branch if certain things are changed?

Reply:
I know we don't want it to be just this single branch, but I'm not sure about adding it to the PR.  I might go along with this only if there is a test for changes to files in .github, and only then it automatically updates the docs.  If that sounds good, I can try to do this before end of sprint.

Reply:
Thanks for the comment. I was not set on my understanding, but I should have phrased my question better. I don't understand exactly, will the target branch be `main`, `develop`, `feature-*` branches as changed, one of them, only `auto-ci-docs-update`? I want to understand what the source/sources of truth are, and also the destination.

Reply:
I'm going to resolved based on a trial of letting it commit to the auto-ci-docs-update branch, and PRing against main.  We can always adjust the target, and automate it more once we are comfortable with what it produces over time.  I adjusted the workflow to execute on PRs opened on a feature-* branch with changes in the .github/workflows directory, so this should only run when the workflows change as a part of a feature.